src/command/remind/index.ts

```diff
@@ -7,7 +7,7 @@ import type { CreateCmdFn } from '../_utils/types.js'
 import { createHelpHandler } from '../_utils/create-help-handler.js'
 
 const $RemindCmdOptions = z.object({
-  at: z.string(),
+  at: z.string().optional(),
```

Comment:
![Screenshot 2023-07-21 at 4 30 17 PM](https://github.com/Runn-Fast/handover/assets/17259943/9b01816b-c09c-4641-95ff-af6e9afa0a51)

Fixing this bug [☝️](https://emojipedia.org/index-pointing-up/)